### DISCUSSION TITLE
会对MOOC课程总结的小聪明

### DISCUSSION THREAD

**Discussion by funcfang** [InternLM/Tutorial#409] [project]
初步计划实现：学生提供 MOOC 课程链接，大模型对该课程涉及的相关课程内容进行总结。
性质：个人学习性项目
成员：自己，个人偏佛系，担心拖队友后腿，所以一人一组
未来可能拓展（理想还是要有的）：对课程内容搭建图数据库（实体识别+关系抽取），根据学生画像进行图数据库检索返回三元组后再次向量相似度排序来实现个性化推荐课程。

**Comment by MING-ZCH** [comment]
看起来是搜索+LLM的track😎
加油~
